src/audio/Kconfig

```diff
@@ -300,6 +300,16 @@ config COMP_TDFB
           for channels selection, channel filter coefficients, and output
           streams mixing.
 
+config COMP_CODEC_ADAPTER
+	bool "Codec adapter"
+	default n
+	help
+	  This component is an adapter between SoF components and any external third
+	  party codecs/libraries. In order to make use of it the library itself should
+	  be statically linked to the SoF FW binary image and the codec details filled in
+	  the codec_lib[] object. It is possible to link several different codecs and use
+	  them in parallel.
+
```

Comment:
Where do we specify path to codec_lib ?

Reply:
Yes good question, I think I will change that description to be a bit more generic.. I think we don't want to provide a complete integration guide here but just highlight few key aspect of codec_adapter usage.